h/static/scripts/test/root-thread-test.js

```diff
@@ -179,6 +185,47 @@ describe('rootThread', function () {
     ]);
   });
 
+  describe('when the thread filter query is set', function () {
+    it('generates a thread filter function to match annotations', function () {
+      fakeBuildThread.reset();
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: 'annotation'});
+
+      rootThread.thread(fakeAnnotationUI.state);
+      var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
+
+      var annotation = {target: [{ selector: {} }]};
+      assert(threadFilterFn({annotation: annotation}));
```

Comment:
This isn't going to stop me merging, but we should try where possible to use precise assertions -- [Chai has lots of them](http://chaijs.com/api/assert/#method_assert) -- because it improves the error messages we get when things go wrong. This should probably be `assert.isTrue`, because you're expecting a literal boolean.